src/gatsby/gatsby-node/pages/createMdxPages.ts

```diff
@@ -242,7 +242,7 @@ const queries = [
         `${featuredJsonPath}/featured-posts.json`,
         JSON.stringify(
           featured.map((node: IBaseQuery) => ({
-            date: new Date(node.frontmatter.date).toISOString(),
+            date: `${node.frontmatter.date} 12:00:00 -0500`,
```

Comment:
Is the intent for this to remain a valid RFC 3339 timestamp? That would look like `${node.frontmatter.date} T12:00:00-05:00` assuming the date is an ISO date.

Also, this will only be office time during standard time, not daylight savings time.